src/js/components/landing/LandingContent.jsx

```diff
@@ -61,6 +61,22 @@ export default class LandingContent extends React.Component {
         })
     }
 
+    checkPermissions(){
```

Comment:
This is the identical to the `checkPermissions()` in NavigationComponent.jsx. Instead of duplicating it, it should be moved a helper and then reused that way.